web/satellite/package-lock.json

```diff
@@ -2226,18 +2226,190 @@
         "yorkie": "^2.0.0"
       },
       "dependencies": {
+        "ansi-styles": {
```

Comment:
since you did not update `package.json`, don't worry about updating `package-lock.json`. You can get the file from the main branch with 
```
git show main:web/satellite/package-lock.json > web/satellite/package-lock.json
```